packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -253,6 +270,7 @@ export async function createInterventionPolicyFromOptimize(
 	modelConfigId: string,
 	optimizeRunId: string
 ): Promise<InterventionPolicy | null> {
+	console.log('Test');
```

Comment:
`Test`

Reply:
IM IN DRAFT! loool
Thanks i was just removing